pysat/__init__.py

```diff
@@ -67,10 +67,9 @@
     os.mkdir(pysat_dir)
     print('Created .pysat directory in user home directory to store settings.')
     # create file with default data directory
+    data_dir = ''
     if (os.environ.get('TRAVIS') == 'true'):
         data_dir = '/home/travis/build/pysatData'
-    else:
-        data_dir = ''
```

Comment:
This change is because this line will never be run on Travis CI.

Reply:
But do we need `data_dir` defined for a non-travis environment?  This could be solved by putting `data_dir = ''` before the if-statement.

Reply:
It is before.  Just forgot to highlight this line.